src/clockcache.c

```diff
@@ -1910,11 +1920,17 @@ clockcache_deinit(clockcache *cc) // IN/OUT
 #endif
    }
 
+   platform_memfrag  memfrag = {0};
+   platform_memfrag *mf      = &memfrag;
    if (cc->lookup) {
-      platform_free(cc->heap_id, cc->lookup);
+      memfrag_init_size(mf, cc->lookup, cc->lookup_size);
+      platform_free(cc->heap_id, mf);
```

Comment:
`free()` needs to be told the fragment's size correctly. This is obtained from the `size` field stashed away when `init()` was done.